[SCRUM-7] Application Wireframe

Marwanyx opened the pull request:
figma design linked

Marwanyx committed:
Create WIREFRAME.MD

Marwanyx committed:
Update WIREFRAME.MD

V-X-L-U reviewed (changes requested):
I left some comments on the Figma page so that you can address them more easily. But overall, I'd say it's a good modern design with a nicely picked colour scheme and the app flow is overall intuitive to anyone who's used any recent mobile apps.

Marwanyx commented:
Updated some pages, I also replied to some of the comments directly on figma (easier), added a couple more pages for extra detail. LMK if its good to go.

V-X-L-U reviewed (approved):
Looks good!
